### DISCUSSION TITLE
arm版本部署milvus-2.4.9，minio能启起来，其他的怎么都启动不起来，

### DISCUSSION THREAD

**Discussion by jhyever** [milvus-io/milvus#38766] [Q&A and General discussion]
1、这是 kubectl logs  my-release-milvus-proxy-7f489c765b-h6scp -n milvus的日志
![image](https://github.com/user-attachments/assets/c5df738d-458e-4e1b-bc80-e4abaebcc624)
![image](https://github.com/user-attachments/assets/f100dfb2-b852-41f1-b902-2083a6bde631)
2、如下是etcd的日志
[root@localhost3 milvus]# kubectl logs  my-release-etcd-0 -n milvus
etcd 02:26:30.29
etcd 02:26:30.29 Welcome to the Bitnami etcd container
etcd 02:26:30.29 Subscribe to project updates by watching https://github.com/bitnami/containers
etcd 02:26:30.29 Submit issues and feature requests at https://github.com/bitnami/containers/issues
etcd 02:26:30.29
etcd 02:26:30.30 INFO  ==> ** Starting etcd setup **
etcd 02:26:30.31 INFO  ==> Validating settings in ETCD_* env vars..
etcd 02:26:30.31 WARN  ==> You set the environment variable ALLOW_NONE_AUTHENTICATION=yes. For safety reasons, do not use this flag in a production environment.
etcd 02:26:30.32 INFO  ==> Initializing etcd
etcd 02:26:30.32 INFO  ==> Generating etcd config file using env variables
etcd 02:26:45.42 INFO  ==>
etcd 02:26:45.42 INFO  ==> get_etcd_active_endpoints:
etcd 02:26:45.43 INFO  ==> Detected data from previous deployments
etcd 02:26:45.44 INFO  ==> The member will try to join the cluster by it's own
etcd 02:26:45.45 INFO  ==> ** etcd setup finished! **

etcd 02:26:45.46 INFO  ==> ** Starting etcd **
{"level":"info","ts":"2024-12-26T02:26:45.494493Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_ADVERTISE_CLIENT_URLS","variable-value":"http://my-release-etcd-0.my-release-etcd-headless.milvus.svc.cluster.local:2379"}
{"level":"info","ts":"2024-12-26T02:26:45.494674Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_AUTO_COMPACTION_MODE","variable-value":"revision"}
{"level":"info","ts":"2024-12-26T02:26:45.494689Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_AUTO_COMPACTION_RETENTION","variable-value":"1000"}
{"level":"info","ts":"2024-12-26T02:26:45.494697Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_AUTO_TLS","variable-value":"false"}
{"level":"info","ts":"2024-12-26T02:26:45.494710Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_CLIENT_CERT_AUTH","variable-value":"false"}
{"level":"info","ts":"2024-12-26T02:26:45.494723Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_DATA_DIR","variable-value":"/bitnami/etcd/data"}
{"level":"info","ts":"2024-12-26T02:26:45.494734Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_ELECTION_TIMEOUT","variable-value":"2500"}
{"level":"info","ts":"2024-12-26T02:26:45.494764Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_HEARTBEAT_INTERVAL","variable-value":"500"}
{"level":"info","ts":"2024-12-26T02:26:45.494785Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_ADVERTISE_PEER_URLS","variable-value":"http://my-release-etcd-0.my-release-etcd-headless.milvus.svc.cluster.local:2380"}
{"level":"info","ts":"2024-12-26T02:26:45.494798Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_CLUSTER","variable-value":"my-release-etcd-0=http://my-release-etcd-0.my-release-etcd-headless.milvus.svc.cluster.local:2380,my-release-etcd-1=http://my-release-etcd-1.my-release-etcd-headless.milvus.svc.cluster.local:2380,my-release-etcd-2=http://my-release-etcd-2.my-release-etcd-headless.milvus.svc.cluster.local:2380"}
{"level":"info","ts":"2024-12-26T02:26:45.494809Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_CLUSTER_STATE","variable-value":"new"}
{"level":"info","ts":"2024-12-26T02:26:45.494816Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_CLUSTER_TOKEN","variable-value":"etcd-cluster-k8s"}
{"level":"info","ts":"2024-12-26T02:26:45.494831Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_LISTEN_CLIENT_URLS","variable-value":"http://0.0.0.0:2379"}
{"level":"info","ts":"2024-12-26T02:26:45.494848Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_LISTEN_PEER_URLS","variable-value":"http://0.0.0.0:2380"}
{"level":"info","ts":"2024-12-26T02:26:45.494860Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_LOG_LEVEL","variable-value":"info"}
{"level":"info","ts":"2024-12-26T02:26:45.494874Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_NAME","variable-value":"my-release-etcd-0"}
{"level":"info","ts":"2024-12-26T02:26:45.494885Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_PEER_AUTO_TLS","variable-value":"false"}
{"level":"info","ts":"2024-12-26T02:26:45.494905Z","caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_QUOTA_BACKEND_BYTES","variable-value":"4294967296"}
{"level":"warn","ts":"2024-12-26T02:26:45.494943Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_TRUSTED_CA_FILE="}
{"level":"warn","ts":"2024-12-26T02:26:45.494955Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_CONF_FILE=/opt/bitnami/etcd/conf/etcd.yaml"}
{"level":"warn","ts":"2024-12-26T02:26:45.494965Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_ON_K8S=yes"}
{"level":"warn","ts":"2024-12-26T02:26:45.494975Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_SNAPSHOTS_DIR=/snapshots"}
{"level":"warn","ts":"2024-12-26T02:26:45.494983Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_BIN_DIR=/opt/bitnami/etcd/bin"}
{"level":"warn","ts":"2024-12-26T02:26:45.494994Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_VOLUME_DIR=/bitnami/etcd"}
{"level":"warn","ts":"2024-12-26T02:26:45.495005Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_CLUSTER_DOMAIN=my-release-etcd-headless.milvus.svc.cluster.local"}
{"level":"warn","ts":"2024-12-26T02:26:45.495016Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_DISASTER_RECOVERY=no"}
{"level":"warn","ts":"2024-12-26T02:26:45.495024Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_KEY_FILE="}
{"level":"warn","ts":"2024-12-26T02:26:45.495030Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_CONF_DIR=/opt/bitnami/etcd/conf"}
{"level":"warn","ts":"2024-12-26T02:26:45.495037Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_DAEMON_GROUP=etcd"}
{"level":"warn","ts":"2024-12-26T02:26:45.495044Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_START_FROM_SNAPSHOT=no"}
{"level":"warn","ts":"2024-12-26T02:26:45.495053Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_INIT_SNAPSHOT_FILENAME="}
{"level":"warn","ts":"2024-12-26T02:26:45.495059Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_INIT_SNAPSHOTS_DIR=/init-snapshot"}
{"level":"warn","ts":"2024-12-26T02:26:45.495066Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_TMP_DIR=/opt/bitnami/etcd/tmp"}
{"level":"warn","ts":"2024-12-26T02:26:45.495076Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_BASE_DIR=/opt/bitnami/etcd"}
{"level":"warn","ts":"2024-12-26T02:26:45.495085Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_CERT_FILE="}
{"level":"warn","ts":"2024-12-26T02:26:45.495092Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_NEW_MEMBERS_ENV_FILE=/bitnami/etcd/data/new_member_envs"}
{"level":"warn","ts":"2024-12-26T02:26:45.495102Z","caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_DAEMON_USER=etcd"}
{"level":"warn","ts":"2024-12-26T02:26:45.495155Z","caller":"embed/config.go:689","msg":"Running http and grpc server on single port. This is not recommended for production."}
{"level":"info","ts":"2024-12-26T02:26:45.495237Z","caller":"etcdmain/etcd.go:73","msg":"Running: ","args":["etcd"]}
{"level":"info","ts":"2024-12-26T02:26:45.495331Z","caller":"etcdmain/etcd.go:116","msg":"server has been already initialized","data-dir":"/bitnami/etcd/data","dir-type":"member"}
{"level":"warn","ts":"2024-12-26T02:26:45.495363Z","caller":"embed/config.go:689","msg":"Running http and grpc server on single port. This is not recommended for production."}
{"level":"info","ts":"2024-12-26T02:26:45.495377Z","caller":"embed/etcd.go:128","msg":"configuring peer listeners","listen-peer-urls":["http://0.0.0.0:2380"]}
{"level":"info","ts":"2024-12-26T02:26:45.495627Z","caller":"embed/etcd.go:136","msg":"configuring client listeners","listen-client-urls":["http://0.0.0.0:2379"]}
{"level":"info","ts":"2024-12-26T02:26:45.495797Z","caller":"embed/etcd.go:311","msg":"starting an etcd server","etcd-version":"3.5.16","git-sha":"f20bbad","go-version":"go1.22.7","go-os":"linux","go-arch":"arm64","max-cpu-set":32,"max-cpu-available":32,"member-initialized":true,"name":"my-release-etcd-0","data-dir":"/bitnami/etcd/data","wal-dir":"","wal-dir-dedicated":"","member-dir":"/bitnami/etcd/data/member","force-new-cluster":false,"heartbeat-interval":"500ms","election-timeout":"2.5s","initial-election-tick-advance":true,"snapshot-count":100000,"max-wals":5,"max-snapshots":5,"snapshot-catchup-entries":5000,"initial-advertise-peer-urls":["http://my-release-etcd-0.my-release-etcd-headless.milvus.svc.cluster.local:2380"],"listen-peer-urls":["http://0.0.0.0:2380"],"advertise-client-urls":["http://my-release-etcd-0.my-release-etcd-headless.milvus.svc.cluster.local:2379"],"listen-client-urls":["http://0.0.0.0:2379"],"listen-metrics-urls":[],"cors":["*"],"host-whitelist":["*"],"initial-cluster":"","initial-cluster-state":"new","initial-cluster-token":"","quota-backend-bytes":4294967296,"max-request-bytes":1572864,"max-concurrent-streams":4294967295,"pre-vote":true,"initial-corrupt-check":false,"corrupt-check-time-interval":"0s","compact-check-time-enabled":false,"compact-check-time-interval":"1m0s","auto-compaction-mode":"revision","auto-compaction-retention":"1µs","auto-compaction-interval":"1µs","discovery-url":"","discovery-proxy":"","downgrade-check-interval":"5s"}
{"level":"info","ts":"2024-12-26T02:26:45.499619Z","caller":"etcdserver/backend.go:81","msg":"opened backend db","path":"/bitnami/etcd/data/member/snap/db","took":"1.838607ms"}
{"level":"info","ts":"2024-12-26T02:26:45.517594Z","caller":"etcdserver/server.go:532","msg":"No snapshot found. Recovering WAL from scratch!"}
{"level":"info","ts":"2024-12-26T02:26:45.543071Z","caller":"etcdserver/raft.go:540","msg":"restarting local member","cluster-id":"9a3dc739eb61eba4","local-member-id":"65c1488df7fa2b50","commit-index":14003}
{"level":"info","ts":"2024-12-26T02:26:45.543897Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"65c1488df7fa2b50 switched to configuration voters=()"}
{"level":"info","ts":"2024-12-26T02:26:45.543995Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"65c1488df7fa2b50 became follower at term 8"}
{"level":"info","ts":"2024-12-26T02:26:45.544034Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"newRaft 65c1488df7fa2b50 [peers: [], term: 8, commit: 14003, applied: 0, lastindex: 14003, lastterm: 8]"}
{"level":"warn","ts":"2024-12-26T02:26:45.547478Z","caller":"auth/store.go:1241","msg":"simple token is not cryptographically signed"}
{"level":"info","ts":"2024-12-26T02:26:45.551809Z","caller":"mvcc/kvstore.go:423","msg":"kvstore restored","current-rev":925}
{"level":"info","ts":"2024-12-26T02:26:45.554517Z","caller":"etcdserver/quota.go:117","msg":"enabled backend quota","quota-name":"v3-applier","quota-size-bytes":4294967296,"quota-size":"4.3 GB"}
{"level":"info","ts":"2024-12-26T02:26:45.557120Z","caller":"etcdserver/server.go:873","msg":"starting etcd server","local-member-id":"65c1488df7fa2b50","local-server-version":"3.5.16","cluster-version":"to_be_decided"}
{"level":"info","ts":"2024-12-26T02:26:45.557326Z","caller":"etcdserver/server.go:773","msg":"starting initial election tick advance","election-ticks":5}
{"level":"info","ts":"2024-12-26T02:26:45.557371Z","caller":"fileutil/purge.go:50","msg":"started to purge file","dir":"/bitnami/etcd/data/member/snap","suffix":"snap.db","max":5,"interval":"30s"}
{"level":"info","ts":"2024-12-26T02:26:45.558056Z","caller":"fileutil/purge.go:50","msg":"started to purge file","dir":"/bitnami/etcd/data/member/snap","suffix":"snap","max":5,"interval":"30s"}
{"level":"info","ts":"2024-12-26T02:26:45.558084Z","caller":"fileutil/purge.go:50","msg":"started to purge file","dir":"/bitnami/etcd/data/member/wal","suffix":"wal","max":5,"interval":"30s"}
{"level":"info","ts":"2024-12-26T02:26:45.558655Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"65c1488df7fa2b50 switched to configuration voters=(3685687020121012590)"}
{"level":"info","ts":"2024-12-26T02:26:45.558584Z","caller":"v3rpc/health.go:61","msg":"grpc service status changed","service":"","status":"SERVING"}
{"level":"info","ts":"2024-12-26T02:26:45.558851Z","caller":"membership/cluster.go:421","msg":"added member","cluster-id":"9a3dc739eb61eba4","local-member-id":"65c1488df7fa2b50","added-peer-id":"332630d13394b96e","added-peer-peer-urls":["http://my-release-etcd-2.my-release-etcd-headless.default.svc.cluster.local:2380"]}
{"level":"info","ts":"2024-12-26T02:26:45.558900Z","caller":"rafthttp/peer.go:133","msg":"starting remote peer","remote-peer-id":"332630d13394b96e"}
{"level":"info","ts":"2024-12-26T02:26:45.558958Z","caller":"rafthttp/pipeline.go:72","msg":"started HTTP pipelining with remote peer","local-member-id":"65c1488df7fa2b50","remote-peer-id":"332630d13394b96e"}
{"level":"info","ts":"2024-12-26T02:26:45.560052Z","caller":"rafthttp/stream.go:169","msg":"started stream writer with remote peer","local-member-id":"65c1488df7fa2b50","remote-peer-id":"332630d13394b96e"}

**Comment by yhmo** [comment]
从proxy的报错上看，它就是连接etcd失败，你可以用etcd client试一下这个etcd是否能正常工作
"use of closed network connection"，比较像这个：https://github.com/kubernetes/kubernetes/issues/125770

**Reply by jhyever** [reply]
It's no use. I submitted a new issue:   https://github.com/milvus-io/milvus/discussions/38771

**Reply by yhmo** [reply]
先把问题简化，部署一个standalone的arm版本看看。
下面这个是其他用户成功部署standalone arm milvus的例子，不妨试一下。
https://blog.csdn.net/weixin_43909848/article/details/128419799

**Reply by jhyever** [reply]
这个部署能部署起来，helm部署不起来集群呢，总是报错

**Reply by jhyever** [reply]
执行命令：kubectl logs my-release-milvus-datanode-7b69675b6d-g7mc4  -n docqa    @yhmo 
 
报如下这个错：
![image](https://github.com/user-attachments/assets/bb2a49c4-f76c-42e9-8c46-b1df288c9938)

 [config/etcd_source.go:52] ["init etcd source"] [etcdInfo="{\"UseEmbed\":false,\"EnableAuth\":false,\"UserName\":\"\",\"PassWord\":\"\",\"UseSSL\":false,\"Endpoints\":[\"my-release-etcd-0.my-release-etcd-headless.docqa.svc.cluster.local:2379\",\"my-release-etcd-1.my-release-etcd-headless.docqa.svc.cluster.local:2379\",\"my-release-etcd-2.my-release-etcd-headless.docqa.svc.cluster.local:2379\"],\"KeyPrefix\":\"by-dev\",\"CertFile\":\"/path/to/etcd-client.pem\",\"KeyFile\":\"/path/to/etcd-client-key.pem\",\"CaCertFile\":\"/path/to/ca.pem\",\"MinVersion\":\"1.3\",\"RefreshInterval\":5000000000}"]
[2024/12/26 08:46:48.863 +00:00] [INFO] [etcd/etcd_util.go:47] ["create etcd client"] [useEmbedEtcd=false] [useSSL=false] [endpoints="[my-release-etcd-0.my-release-etcd-headless.docqa.svc.cluster.local:2379,my-release-etcd-1.my-release-etcd-headless.docqa.svc.cluster.local:2379,my-release-etcd-2.my-release-etcd-headless.docqa.svc.cluster.local:2379]"] [minVersion=1.3]
[2024/12/26 08:46:53.864 +00:00] [INFO] [paramtable/base_table.go:209] ["init with etcd failed"] [error="context deadline exceeded"]
[2024/12/26 08:46:53.865 +00:00] [INFO] [paramtable/component_param.go:4124] ["DeployModeEnv is not set, use default"] [default=0.5]
[2024/12/26 08:46:53.866 +00:00] [INFO] [paramtable/hook_config.go:21] ["hook config"] [hook={}]
[2024/12/26 08:46:53.866 +00:00] [DEBUG] [config/refresher.go:67] ["start refreshing configurations"] [source=FileSource]
[2024/12/26 08:46:53.866 +00:00] [INFO] [logutil/logutil.go:163] ["Log directory"] [configDir=]
[2024/12/26 08:46:53.866 +00:00] [INFO] [logutil/logutil.go:164] ["Set log file to "] [path=]
[2024/12/26 08:46:53.867 +00:00] [INFO] [roles/roles.go:277] [setupPrometheusHTTPServer]
[2024/12/26 08:46:53.867 +00:00] [INFO] [http/server.go:152] ["management listen"] [addr=:9091]
[2024/12/26 08:46:53.868 +00:00] [INFO] [gc/gc_tuner.go:137] ["GC Helper initialized."] ["Initial GoGC"=100] [minimumGOGC=30] [maximumGOGC=200] [memoryThreshold=59355109785]
[2024/12/26 08:46:53.869 +00:00] [INFO] [etcd/etcd_util.go:47] ["create etcd client"] [useEmbedEtcd=false] [useSSL=false] [endpoints="[my-release-etcd-0.my-release-etcd-headless.docqa.svc.cluster.local:2379,my-release-etcd-1.my-release-etcd-headless.docqa.svc.cluster.local:2379,my-release-etcd-2.my-release-etcd-headless.docqa.svc.cluster.local:2379]"] [minVersion=1.3]
[2024/12/26 08:46:58.870 +00:00] [ERROR] [datanode/service.go:250] ["failed to connect to etcd"] [error="context deadline exceeded"] [stack="github.com/milvus-io/milvus/internal/distributed/datanode.(*Server).init\n\t/workspace/source/internal/distributed/datanode/service.go:250\ngithub.com/milvus-io/milvus/internal/distributed/datanode.(*Server).Run\n\t/workspace/source/internal/distributed/datanode/service.go:188\ngithub.com/milvus-io/milvus/cmd/components.(*DataNode).Run\n\t/workspace/source/cmd/components/data_node.go:55\ngithub.com/milvus-io/milvus/cmd/roles.runComponent[...].func1\n\t/workspace/source/cmd/roles/roles.go:121"]

**Reply by yhmo** [reply]
还是etcd的问题，换成那个成功的例子里的etcd镜像，查看etcd的日志

**Reply by jhyever** [reply]
报另外一个问题了：  @yhmo 
{"level":"warn","ts":"2024-12-26T09:06:35.707613Z","caller":"etcdserver/server.go:2139","msg":"failed to publish er to cluster through raft","local-member-id":"65c1488df7fa2b50","local-member-attributes":"{Name:my-release-etcdRLs:[http://my-release-etcd-0.my-release-etcd-headless.docqa.svc.cluster.local:2379]}","request-path":"/0/members7fa2b50/attributes","publish-timeout":"10s","error":"etcdserver: request timed out"}
{"level":"warn","ts":"2024-12-26T09:06:35.714845Z","caller":"rafthttp/probing_status.go:68","msg":"prober detectey status","round-tripper-name":"ROUND_TRIPPER_SNAPSHOT","remote-peer-id":"332630d13394b96e","rtt":"0s","error":"dookup my-release-etcd-2.my-release-etcd-headless.default.svc.cluster.local on 10.96.0.10:53: no such host"}
{"level":"warn","ts":"2024-12-26T09:06:35.714943Z","caller":"rafthttp/probing_status.go:68","msg":"prober detectey status","round-tripper-name":"ROUND_TRIPPER_SNAPSHOT","remote-peer-id":"c0900869b5a892b2","rtt":"0s","error":"dookup my-release-etcd-1.my-release-etcd-headless.default.svc.cluster.local on 10.96.0.10:53: no such host"}
{"level":"warn","ts":"2024-12-26T09:06:35.714982Z","caller":"rafthttp/probing_status.go:68","msg":"prober detectey status","round-tripper-name":"ROUND_TRIPPER_RAFT_MESSAGE","remote-peer-id":"c0900869b5a892b2","rtt":"0s","errorp: lookup my-release-etcd-1.my-release-etcd-headless.default.svc.cluster.local on 10.96.0.10:53: no such host"}
{"level":"warn","ts":"2024-12-26T09:06:35.715007Z","caller":"rafthttp/probing_status.go:68","msg":"prober detectey status","round-tripper-name":"ROUND_TRIPPER_RAFT_MESSAGE","remote-peer-id":"332630d13394b96e","rtt":"0s","errorp: lookup my-release-etcd-2.my-release-etcd-headless.default.svc.cluster.local on 10.96.0.10:53: no such host"}
{"level":"info","ts":"2024-12-26T09:06:36.691131Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"652b50 is starting a new election at term 8"}

**Reply by jhyever** [reply]
![image](https://github.com/user-attachments/assets/5be95049-fcb1-4151-8294-7e65f88be4a3)

![image](https://github.com/user-attachments/assets/b76b849e-32d1-4f7a-9510-5c12ca04b0e5)

**Reply by yhmo** [reply]
"errorp: lookup my-release-etcd-2.my-release-etcd-headless.default.svc.cluster.local on 10.96.0.10:53: no such host"
这种报错应该是跟k8s里的配置相关，你们那边如果是有devops人员能搞定这种问题最好，不然类似问题会很多。我们这边仅仅看报错信息也不知具体是哪里配错。

**Reply by jhyever** [reply]
现在etcd启动起来了，其他的报了这样的错    @yhmo 
kubectl logs  -n docqa my-release-milvus-proxy-7d9cc56b9d-rspmj![image](https://github.com/user-attachments/assets/8963a922-7da6-4b84-a3e1-f74cd39801af)

![image](https://github.com/user-attachments/assets/48d31cba-adbc-4d99-b1c4-4c2ce1308ce0)

![image](https://github.com/user-attachments/assets/a51f5f52-657c-4c56-9081-b31e1a684c14)

**Reply by yhmo** [reply]
看mixcoord的日志

**Reply by jhyever** [reply]
已经解决了，感谢

**Reply by yangyangdy** [reply]
请问是怎么解决的，遇到了类似的问题

**Reply by xiaofan-luan** [reply]
> 请问是怎么解决的，遇到了类似的问题

1. 建议新开个issue，上传你的日志方面我们排查。
2.建议不要再用2.4.9了，用2.5.10
